doc/transform_methods/hansenlaw.rst

```diff
@@ -9,52 +12,126 @@ The Hansen and Law transform is the work of E. W. Hansen and P.-L. Law [1].
+the noisy data case.*
 
-The key advantage of the algorithm is its computational simplicity that amounts to only a few lines of code. 
+The key advantage of the algorithm is its computational simplicity that 
+amounts to only a few lines of code. 
 
 
 
 How it works
 ------------
 
-.. image:: https://cloud.githubusercontent.com/assets/10932229/13250293/40a333c2-da7d-11e5-9647-d8404a12626a.png
-   :width: 650px
-   :alt: Path of integration
+.. figure:: https://cloud.githubusercontent.com/assets/10932229/13543157/c83d3796-e2bc-11e5-9210-12be6d24b8fc.png
+   :width: 200px
+   :alt: projection diag
+   :align: right
+   :figclass: align-center
+
+   Projection geometry (Fig. 1 [1])
+
+image function |nbsp|  :math:`f(r)`, |nbsp| projected function |nbsp|  :math:`g(R)`
+
+forward Abel transform 
+
+.. math:: g(R) = 2 \int_R^\infty \frac{f(r) r}{\sqrt{r^2 - R^2}} dr 
+
+inverse Abel transform 
+
+.. math:: f(r) = -\frac{1}{\pi}  \int_r^\infty \frac{g^\prime(R)}{\sqrt{R^2 - r^2}} dR
+
+
+
+The Hansen and Law method makes use of a coordinate transformation, which is 
+used to model the Abel transform, and derive *reconstruction* filters. The Abel
+transform is treated as a system modeled by a set of linear differential 
+equations. In this framework the forward Abel transform :math:`g(R)` is 
+the solution of a differential equation with :math:`f(r)` as its driving 
+function. Similarly, the Abel inversion :math:`f(r)` is a solution of a 
+differential equation with :math:`g^\prime(R)` as its driving function. 
+
+.. figure:: https://cloud.githubusercontent.com/assets/10932229/13544803/13bf0d0e-e2cf-11e5-97d5-bece1e61d904.png 
+   :width: 350px
+   :alt: recursion
+   :align: right
+   :figclass: align-center
+
+   Recursion: pixel value from adjacent outer-pixel
+
+
+forward transform
 
-The Hansen and Law method makes use of a coordinate transformation, which is used to model the Abel transform, and derive *reconstruction* filters. The Abel transform is treated as a system modeled by a set of linear differential equations. 
+.. math:: 
 
-.. image:: https://cloud.githubusercontent.com/assets/10932229/13251144/88f7a1d0-da82-11e5-8c09-7bf2dc4be830.png
-   :width: 550px
-   :alt: hansenlaw-iteration
+  x_{n+1} &= \Phi_n x_n + \Gamma_n f_n 
 
-The algorithm iterates along each row of the image, starting at the out edge, and ending at the center.
+  g_n &= \tilde{C} x_n
+
+inverse transform
+
+.. math:: 
+
+  x_{n+1} &= \Phi_n x_n + \Gamma_n g^\prime_n 
+
+  f_n &= \tilde{C} x_n
+
+
+Note the only difference between the *forward* and *inverse* algorithms is 
+the exchange of :math:`f_n` with :math:`g^\prime_n` (or :math:`g_n`).
+
+|
+|
+
+:math:`\Phi_n` and :math:`\Gamma_n` are functions with predetermined 
+parameter constants, all listed in [1].
+
+
+The algorithm iterates along each individual row of the image, starting at 
+the out edge, ending at the center-line. This provides for efficient code 
+parallelization.
```

Comment:
Good text, tks. 

 Philosophically, I find this aspect of Abel transforms amazing:
>all rows in a column should be independent should actually be true of all abel transform methods.

Each row corresponds to a definite theta, that the reconstruction does not care about the other rows in an image was hard to get my head around.